marimbabot_ur5_moveit_config/config/arm_controller_ur5.yaml

```diff
@@ -17,12 +17,12 @@ pos_joint_traj_controller:
   constraints:
     goal_time: 0.6
     stopped_velocity_tolerance: 0.05
-    ur5_shoulder_pan_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_shoulder_lift_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_elbow_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_1_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_2_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_3_joint: {trajectory: 0.1, goal: 0.01}
+    ur5_shoulder_pan_joint: {trajectory: 0.1, goal: 0.1}
```

Comment:
Why did you change these values?

Reply:
Current PID values make it impossible to follow the trajectory. This is just for simulation to work at the moment, will be changed.